app/bundles/UserBundle/Controller/UserController.php

```diff
@@ -220,6 +220,11 @@ public function editAction(Request $request, LanguageHelper $languageHelper, Use
         \assert($model instanceof UserModel);
         $user = $model->getEntity($objectId);
 
+        /** @var \Mautic\CoreBundle\Model\AuditLogModel $auditLogModel */
+        $auditLogModel      = $this->getModel('core.auditlog');
```

Comment:
I'd suggest to add the audit log model to the method params. It will be auto-wired. It's a cleaner way to get it.